### ISSUE TITLE
WCP: Color contrast on https://adaptivecards.io/ links

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6327]
### Target Platforms

NodeJS

### SDK Version

Website

### Application Name

https://adaptivecards.io/

### Problem Description

[WCP scan result](https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanReport?siteId=a9e04ff4-f488-e711-80c3-0004ffc0ff2c&urlId=10351)

**Paths:** 
```
a[href$="storybook\.js\.org"]
a[href$="hexo\.io"]
```
**Snippets**:
```
<a class="ac-footer-link-no-chevron" target="_blank" rel="noopener" href="https://storybook.js.org">Storybook.js</a>
<a class="ac-footer-link-no-chevron" target="_blank" rel="noopener" href="https://hexo.io">Hexo</a>
```
**How to Fix:**
Fix the following:

- Element has insufficient color contrast of 4.06 (foreground color: #0b78d0, background color: #f2f2f2, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1Element has insufficient color contrast of 4.06 (foreground color: #0b78d0, background color: #f2f2f2, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

- Use foreground color: #096cbb and the original background color: #f2f2f2 to meet a contrast ratio of 4.84:1Use foreground color: #096cbb and the original background color: #f2f2f2 to meet a contrast ratio of 4.84:1

- Or use background color: #fefefe and the original foreground color: #0b78d0 to meet a contrast ratio of 4.52:1.

### Card JSON

```JSON
website
```
